### ISSUE TITLE
[Obsolete] MONDO:0850093 'absence epilepsy'

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#8468]
**Mondo term (ID and Label)**
MONDO:0850093 'absence epilepsy'

**Reason for deprecation**
This is a phenotype. Discussed at [epilepsy workshop](https://www.youtube.com/watch?v=W_WC7QZsL0M) around minute 18.

**Suggested term to consider**
[HPO](https://hpo.jax.org/browse/term/HP:0011147)

**Comment by twhetzel** [comment]
@nicolevasilevsky the [PR](https://github.com/monarch-initiative/mondo/pull/8469) is approved and merged. Does this issue also need a tag of 'on the list' or anything else?

**Comment by nicolevasilevsky** [comment]
yes it does, I added it, thanks!
